templateflow/conf/__init__.py

```diff
@@ -76,6 +75,11 @@ def update_home(force=False):
     return False
 
 
-TF_LAYOUT = Layout(
-    str(TF_HOME), validate=False, config='templateflow',
-    ignore=['.git', '.datalad', '.gitannex', '.gitattributes', 'scripts'])
+TF_LAYOUT = None
+try:
+    from .bids import Layout
+    TF_LAYOUT = Layout(
+        str(TF_HOME), validate=False, config='templateflow',
+        ignore=['.git', '.datalad', '.gitannex', '.gitattributes', 'scripts'])
+except ImportError:
```

Comment:
What condition should this happen under? You've got `pybids ~= 0.9.2` in your `install_requires`.

Reply:
For some reason pybids is not available when running the update script during install - could dig more into this, but I'm not sure it's worthy.

Reply:
Ah, you could add pybids to `setup_requires` and `pyproject.toml`. Very heavy lifts, though...